scripts/lb-wrapper

```diff
@@ -81,7 +82,7 @@ log "Info" "Using yt-dlp $(yt-dlp --version)"
 if [[ $XKLB_INTERNAL_CMD == "tubeadd" ]]; then
     xklb_full_cmd="lb tubeadd '${XKLB_DB_FILE}' '${URL_OR_SEARCH_TERM}' --force ${VERBOSITY}"
 elif [[ $XKLB_INTERNAL_CMD == "dl" ]]; then
-    xklb_full_cmd="lb dl '${XKLB_DB_FILE}' --video --search '${URL_OR_SEARCH_TERM}' ${FORMAT_OPTIONS} --write-thumbnail --subs --live --live-from-start -o '${OUTTMPL}' ${VERBOSITY}"
+    xklb_full_cmd="lb dl '${XKLB_DB_FILE}' --video --search '${URL_OR_SEARCH_TERM}' ${FORMAT_OPTIONS} --write-thumbnail --subs --live --live-from-start -o='${OUTTMPL}' ${VERBOSITY}"
```

Comment:
To make the problem more clear it might make sense to add a note _in the code_ and link to this PR or the referenced issue.

nitpick: I would pull the other yt-dlp options into there as well: `${FORMAT_OPTIONS} --write-thumbnail --live --live-from-start -o='${OUTTMPL}'` maybe into a new variable `YTDLP_OPTIONS`

Reply:
Great suggestion from @chapmanjacobd:

@ZachLiebl can you consider creating a constant named something like YT_DLP_OPTIONS on around Line 26, that ingests (concatenates) both ${FORMAT_OPTIONS} and ${OUTTMPL} ?